src/components/EditListingWizard/EditListingWizard.js

```diff
@@ -28,8 +28,8 @@ import css from './EditListingWizard.css';
 // Show availability calendar only if environment variable availabilityEnabled is true
 const availabilityMaybe = config.enableAvailability ? [AVAILABILITY] : [];
 
-// TODO: PHOTOS panel needs to be the last one since it currently contains PayoutDetailsForm modal
-// All the other panels can be reordered.
+// You can reorder panels by reordering tabs in this array.
+// If you change the order of the panels, remember to change the texts in buttons accordinly from en.json file (keys starting with EditListingWizard.saveNew...)
```

Comment:
It's probably better to do this change separate PR. (I have a feeling that there are other issues that might need changes too - but I'm not sure...)